src/index.ts

```diff
@@ -164,4 +167,7 @@ export {
     HeaderPredicateFactory,
     MappingHeader,
     withJsxExport,
+    MeasureNumberFormat,
+    MeasureNumberFormatType,
+    IPositioning,
```

Comment:
let's export also `IFormatPreset` so we can use it in AD

Reply:
`IFormatPreset` is already exported within `MeasureNumberFormatType`. The question is, whether this is not a weird way.